.gitmodules

```diff
@@ -10,7 +10,7 @@
 	ignore = untracked
 [submodule "external/calendar"]
 	path = external/calendar
-	url = https://github.com/Etar-Group/android_frameworks_opt_calendar.git
+	url = https://github.com/mikeNG/android_frameworks_opt_calendar.git
```

Comment:
can you drop this change? We would like to keep our own version. Feel free to send patches as PRs to that repo.

Reply:
Yes I intend to drop it before merging, it's only temporary to make it easier to test this, build will fail otherwise.

https://github.com/Etar-Group/android_frameworks_opt_calendar/pull/1